README.md

```diff
@@ -52,7 +41,12 @@ Vault will have:
 - a map of Google service account keys at `/credentials/google/${serviceAccountEmail}`
 - a map of AWS IAM keys at `/credentials/amazon/${accountName}`
 - common provider specific data shared amongst clusters at `/clusters/common/${provider}`
-- a map of clusters `/clusters/${environment}/${clusterName}` containing all sensitive information specific to a single cluster
+- a `/clusters/all` record whose value is a map of all `clusterSlug`s to their path in Vault, eg `mycluster: '/clusters/production/mycluster'`
+- a map of clusters `/clusters/${environment}/${clusterSlug}` containing all information specific to a single cluster.  The record will have
+  + A `value` key with a JSON blob of general cluster info, as defined by the schema.
+  + A `channels` key with a JSON array of all channels the cluster belongs to
+- a `/channels/all` record whose `value` is a comma-separated list of all channels
```

Comment:
should this be a JSON array for consistency?

Reply:
Yep, that was easier to do internally as well.